src/main/java/kernbeisser/Windows/LogIn/SimpleLogIn/SimpleLogInView.java

```diff
@@ -59,6 +58,7 @@ public void initialize(SimpleLogInController controller) {
                 Tools.showUnexpectedErrorWarning(e);
             }
         }
+        Tools.activateBarcodeListener();
```

Comment:
This is actually the wrong place the better place would be in the main method or in the controller of the login

Reply:
I didn't see the comment above. This is only test code so leave it like this